terraform/azure/bin/ovpn-client-config

```diff
@@ -0,0 +1 @@
+ssh -t "ubuntu@$(terraform output bastion.ip)" sudo docker run --volumes-from ovpn-data --rm gosuri/openvpn ovpn_getclient "${1}" > "${1}-apollo.ovpn"
```

Comment:
I think we can move the code to a common folder because it exact copy of that which we use for aws